migration-engine/connectors/sql-migration-connector/src/sql_database_step_applier.rs

```diff
@@ -104,7 +104,7 @@ impl DatabaseMigrationStepApplier for SqlMigrationConnector {
     #[tracing::instrument(skip(self, script))]
     async fn apply_script(&self, migration_name: &str, script: &str) -> ConnectorResult<()> {
         self.host
-            .print(&format!("Applying migration `{}`", migration_name))
+            .print(&format!("Applying migration `{}`\n", migration_name))
```

Comment:
```suggestion
            .print(&format!("Applying migration `{}`", migration_name))
```
@tomhoule let's unblock for now and see we can see what we want to do later here